util-node/getPaginationPath.js

```diff
@@ -0,0 +1,9 @@
+function getPaginationPath(pagePath, nodeName) {
```

Comment:
This file doesn't seem to be used by any node utility (e.g.: `gatsby-node.js`) so it should stay on `./src/utils` not `./util-node`

Reply:
Note, you could honestly embed this within `gatsby-node.js` as we do the same there

Reply:
Yeah, but i need this function in Pagination component.

Reply:
and it use by gatsby-node.js
`const getBlogPagePath = getPaginationPath(blogPath, node.name);` [example](https://github.com/nodejs/nodejs.dev/pull/2951/files#diff-a6f3904c41211be170d9b0eb87e49ea8d02d0fd5a7224d6772174b425dacf90fR172)